packages/ui/src/components/atoms/Switch/Switch.tsx

```diff
@@ -30,7 +30,7 @@ const Switch = ({isEnabled, handleToggle, style}: SwitchProps) => {
   const Colors = useThemeColor();
 
   const toggleSwitch = () => {
-    handleToggle(!isEnabled);
+    handleToggle != null && handleToggle(!isEnabled);
```

Comment:
It's weird to do that because handleToggle is required so it shouldn't be null, use readonly props of parent component instead